src/components/heroBanner/HeroBanner.styled.js

```diff
@@ -0,0 +1,77 @@
+import { styled } from "styled-components";
+import { StyledButton } from "../button/Button.styled";
+
+export const LeftStyledHeroBanner = styled.div`
+    height: 45rem;
+    position: relative;
+    display: flex;
+    justify-content: center;
+
+    h2 {
+      margin: 0px;
+    }
+
+    p {
+      margin-top: .75rem;
+      margin-bottom: 1.25rem;
+      gap: .625rem;
```

Comment:
is the `gap` currently used for anything? `gap` only works if it also has `display: flex` or `display: grid`